src/db/tables/email_addresses.js

```diff
@@ -54,6 +54,20 @@ async function addSubscriberUnverifiedEmailHash (user, email) {
 
 async function resetUnverifiedEmailAddress (emailAddressId) {
   const newVerificationToken = uuidv4()
+
+  // Time in ms to require between verification reset.
+  const verificationWait = 5 * 60 * 1000 // 5 minutes
```

Comment:
Interesting tradeoff to consider, what if we make this into a string

`${5 * 50 * 1000} MILLISECOND`

and pass that directly into the whereRaw below, is that cleaner? 2x calculations vs. 1 here?

Reply:
Hm yeah I actually did something like that originally (I think I put `5 MINUTES` in the constant, Postgres has a very flexible parser for intervals.

I thought we might end up moving that constant out of the function, or making it an argument etc, so I worried about a future JS developer seeing something that isn't a `Number` representing milliseconds in a constant and getting confused. Maybe I'm overthinking.

Reply:
I mean one thing we could do here is just forget about using a parameter and just hardcode `INTERVAL 15 MINUTES`? Unless we plan to pull the constant out of this function then it probably is clearer anyway.

Reply:
Got it.. hmm I think the current way is better than `INTERVAL 15 MINUTES` :D